src/osd/osd_types.cc

```diff
@@ -431,14 +457,26 @@ void osd_stat_t::encode(bufferlist &bl, uint64_t features) const
   encode(num_shards_repaired, bl);
   encode(num_osds, bl);
   encode(num_per_pool_osds, bl);
```

Comment:
need to bring in the num_per_pool_omap_osds field *before* this patch to avoid breaking encoding compatibility

Reply:
@liewegas Switched to version 14 for compatibility, handling of version 13 for num_per_pool_omap_osds.

Reply:
Perfect!